ios/RCTFBSDK/share/RCTConvert+FBSDKSharingContent.m

```diff
@@ -132,15 +128,16 @@ static void RCTAppendGenericContent(RCTFBSDKSharingContent contentObject, NSDict
 static FBSDKShareOpenGraphAction *RCTBuildOpenGraphAction(NSDictionary *actionData)
 {
   FBSDKShareOpenGraphAction *action = nil;
+  NSString *actionType = nil;
```

Comment:
I don't think we need to declare this variable here, could just be `NSString *actionType = [RCTConvert...` on line 133

Reply:
okay. i will improve this in the preview photo pr :)